services/nodeping/nodeping-status.tester.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('NodePing status - up (live)')
+  .get('/jkiwn052-ntpp-4lbb-8d45-ihew6d9ucoei.json')
+  .expectJSON({
+    name: 'Status',
+    value: 'up',
+  })
+
+t.create('NodePing status - up (mock)')
+  .get('/jkiwn052-ntpp-4lbb-8d45-ihew6d9ucoei.json')
+  .intercept(nock =>
+    nock('https://nodeping.com')
+      .get(
+        '/reports/results/jkiwn052-ntpp-4lbb-8d45-ihew6d9ucoei/1?format=json'
+      )
+      .reply(200, [{ su: true }])
+  )
+  .expectJSON({
+    name: 'Status',
+    value: 'up',
+  })
+
+t.create('NodePing status - down (live)')
+  .get('/wf47q49c-zdmk-4ngx-88g4-wa7mzgluhhaf.json')
+  .expectJSON({
+    name: 'Status',
+    value: 'down',
```

Comment:
How confident are we that this will _always_ be down and that jkiwn052-ntpp-4lbb-8d45-ihew6d9ucoei will _always_ be up? 

I'm thinking it might be better to do a `withRegex` to ensure the value is `up` or `down` on these two tests to avoid any future test failures if the status for either of those changes briefly

Reply:
> The "up" test is on @NodePing, so if it is down, every other NodePing test will also be down. They have a pretty good track record: if you look at the [human-readable version](https://nodeping.com/reports/uptime/jkiwn052-ntpp-4lbb-8d45-ihew6d9ucoei) for this checkUuid, you can see that they've only had 10 minutes of downtime in the past 2 years.

> The "down" report is pinging a server that doesn't exist. I can't imagine how it could start working. It is in my personal account, but I'm a very satisfied NodePing customer (they are great, especially if you are cost-sensitive and have a zillion microservices and domains). I could just delete this test, since the mocked results should still get us 100% code coverage. Let me know if you want me to.

You can delete one of those tests (remember the live tests are more like integration/functional tests, not unit tests for code coverage purposes), but please change the validation to the set of allowed values vs. an explicit value to be consistent with the other tests. 

The easiest way to do this is probably via `Joi` by importing it:

```
const Joi = require('joi')
```

And then update the expect to:
```
.expectJSONTypes(
  Joi.object().keys({
    name: 'Status',
    value: Joi.equal('up', 'down').required(),
  })
)
```